### ISSUE TITLE
Locale parameter for i18n chatbox

### ISSUE THREAD

**Issue by rsereir** [walterholohan/react-native-crisp-chat-sdk#118]
Hello,

I see on README and on closed issue "locale" setting is not supported. I see on web sdk locale is supported with a setLocale method. Someone know how if it would be possible to interact with this setLocale api ?

I think this would be a good update, but I don't really understand the technical blockage that was encountered on the first attempt

Glad to have your feedback, thank you

**Comment by Acetyld** [comment]
Yup with expo changing in expo is not that easy @walterholohan or is it?
